### ISSUE TITLE
Come up with proposal how to facilitate instances getting ideas/requests into Papercuts Process 

### ISSUE THREAD

**Issue by jaycmb** [openfoodfoundation/product-pipe#40]

**Comment by RachL** [comment]
Draft is stored here: https://docs.google.com/presentation/d/1TQTn5fkfG-2JpBR5OV-p8Q-1FG6PhIvIFcj0z76y2Cw/edit#slide=id.g47c412a157_2_8 

Needing a second iteration before sharing with the whole product team

**Comment by jaycmb** [comment]
Input from today´s discussion in product circle needs to be included 

General Feedback

- sounds like a good proposal, to have all feature requests in one place instead of discourse for wishlist and Github for papercuts

**Concerns:**

- How are people going to use GitHub? Will it create more problems and support needs?

→ Eriol proposed: GitHub Buddies

Discussed improvements

- to kick-off the process having an initial call how to create issues on giithub with new wishlist template

Still pending:

- Prioritisation once wishlist items made it into inception pipe → Proposal pending, blocked by Product Vision Process (connected to identfied themes)

**Comment by jaycmb** [comment]
Feedback from product circle meeting has been adressed, see https://docs.google.com/presentation/d/1TQTn5fkfG-2JpBR5OV-p8Q-1FG6PhIvIFcj0z76y2Cw/edit?usp=sharing 

Next steps, as discussed today

1. Revise "Wishlist" Template 
- [x] Create new template (easier to fill out and more general, both usable for papercuts and feature candidates)
- [ ] -> @RachL Rachel to update

2. Communicate new process

- [ ] Briefly update in product circle and discuss if there is an urge to get the new process in place?)
- [ ] Discourse Post: tbd if **after** proposals for _Product Vision, Prioritised Themes_ and _Feature Candidate Prioritisation process_ (once wishlist items made it into inception pipe) got out for feedback?

3. Implementation
- [ ] Set up initial call/sessions to walk Instance Managers + Customer Support People through the new process, template and exemplary create new issue
